scripts/testing/load/Dockerfile

```
@@ -81,6 +81,7 @@ ENV PATH=$PATH:/usr/local/go/bin
 
 RUN mv /agent/test/config/agent/nginx-agent-otel-load.conf /agent/test/load/nginx-agent.conf
 RUN mkdir /var/run/nginx-agent/
+RUN mkdir /var/log/nginx-agent/
```

Can change to a single line?
```suggestion
RUN mkdir /var/log/nginx-agent/ /var/log/nginx-agent/
```